INTEGRATION_TEST.md

```diff
@@ -1,39 +1,31 @@
 # Integration Test Setup
 
-The travis build is configured to test against an EXASolution database instance hosted by EXASOL. This text describes how to get such a test DB, as well as how to make the right entries in the .travis.yml file.
+Integration testing is done by GitHub Actions contained in this repository, which provide a CI/CD pipeline to test, build, and deploy sqlalchemy_exasol to Pypi.
 
-## 1 - Register with EXASOL
+Two main workflows are used for this purpose:
 
-You need an account on the EXASOL user portal. Sign up here: https://www.exasol.com/portal/display/WEL/Home
+## CI
 
-## 2 - Request a Demo User
+This is meant to be used as the test workflow. It's located under:
 
-In the issue tracker (https://www.exasol.com/support/secure/Dashboard.jspa) create a ticket on Project "Public Demo" with Type "Registration". Fill in whatever fields are required. You will receive an e-mail with an automatically generated user account with test schemata in an EXASolution 4.x and 5.x database cluster (as of January 2015). 
+	sqlalchemy_exasol/.github/workflows/CI.yml
 
-The SQLAlchemy test suite requires the schema 'test_schema' for all tests to pass. So you will have to ask EXASOL to grant your user access to this schema.
+This workflow will be executed anytime there's a commit pushed to any branch that is **not master**, or whenever there's a pull request created where the base branch is **not master**. It runs a Docker Container with an Exasol databse for every version specified in *matrix.exasol_version*, and uses those DBs to executes all tests in the repository. If everything went fine, it will create a package distribution using both sdist and wheel.
```

Comment:
```suggestion
This workflow will be executed anytime there's a commit pushed to any branch that is **not master**, or whenever there's a pull request created where the base branch is **not master**. It runs a Docker Container with an Exasol database for every version specified in *matrix.exasol_version*, and uses those DBs to executes all tests in the repository. If everything went fine, it will create a package distribution using both sdist and wheel.
```